frontend/src/components/widgets/inputs/CurveInput.svelte

```diff
@@ -234,7 +234,6 @@
 			fill: var(--color-1-nearblack);
 			stroke: var(--color-e-nearwhite);
 			stroke-width: 0.01;
-			r: 0.02;
```

Comment:
Ah, it looks like this isn't a valid CSS property pre-SVG2. But it will be when/if browsers ever support SVG2. But I think it's still working as intended (at least in Chrome) with this property applied? I don't think removing it is okay without adding it back in the SVG circle elements as a proper attribute. Can you please add it to the relevant `<circle>`s?